Slider/Assets/Scripts/Map/Mountain/Minecart/Minecart.cs

```diff
@@ -149,6 +152,19 @@ public override STile DropItem(Vector3 dropLocation, System.Action callback=null
 
     #endregion
 
+
+    private void AddTracker()
```

Comment:
this should prob just be a `GetTrackerSprite()` and then add the tracker in a single line